packages/monitoring-system/metrics-dashboard/cloudbuild.yaml

```diff
@@ -0,0 +1,32 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+steps:
+  - name: gcr.io/cloud-builders/gcloud
+    id: "get-firebase-token"
+    waitFor: ["-"]
+    dir: './packages/monitoring-system/metrics-dashboard'
+    entrypoint: 'bash'
+    args: 
+      - '-c'
+      - "gcloud secrets versions access latest --secret=firebase-token-metrics-dashboard --format='get(payload.data)' | tr '_-' '/+' | base64 -d > firebase-token.txt"
```

Comment:
You should be able to put each of these on its own line

Reply:
are you sure? I thought the whole line was passed as a string to bash because of the `-c` flag

Reply:
I can move it to a separate `.sh` file if you want, that might be cleaner

Reply:
PS: The official docs have the whole line in one arg - https://cloud.google.com/cloud-build/docs/securing-builds/use-encrypted-secrets-credentials